docker/amdvlk.Dockerfile

```diff
@@ -87,6 +95,10 @@ RUN EXTRA_FLAGS="" \
     && cmake --build . --target not
 
 # Run the lit test suite.
-RUN cmake --build . --target check-amdllpc -- -v \
+RUN if echo "$FEATURES" | grep -q "+sanitizer" ; then \
+        export ASAN_OPTIONS=detect_leaks=0 \
+        && export LD_PRELOAD=/usr/lib/llvm-9/lib/clang/9.0.0/lib/linux/libclang_rt.asan-x86_64.so; \
```

Comment:
Can we move this up to the feature test that sets EXTRA_FLAGS and add empty defaults for both ASAN_OPTIONS and LD_PRELOAD?

Reply:
How should it look like? Iirc, `export`s are lost between run commands, so we have to set them either in the RUN directive that runs the `check-` commands or use docker `ENV` directives (though I don’t know how they can be conditional).

Reply:
Ah, you are right. Let's not over-complicate this.